recipes/fio_py/meta.yaml

```diff
@@ -0,0 +1,41 @@
+package:
+  name: fio_py
+  version: 1.0
+
+source:
+  git_url: https://github.com/smithsp/fusion-io.git
```

Comment:
Please use ``tarballs`` instead of ``git_url``
Please see
https://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=tarballs#build-from-tarballs-not-repos